deploy/ccx-notification-service.yaml

```diff
@@ -32,6 +32,8 @@ objects:
     failedJobsHistoryLimit: 3
     jobTemplate:
       spec:
+        # In PROD 1 cronjob takes around 45 min. Deadline set to 60 mins.
```

Comment:
seems a tad close to the limit, isn't there a better way to kill jobs right after they finish?
also, what happens when it gets killed before finishing?

Reply:
@Bee-lee the job gets killed only when it takes more than 1 hour to run. Then it is marked as failed and a new one will start based on the schedule.
https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-termination-and-cleanup